### DISCUSSION TITLE
FBA Recommended Inventory Qty

### DISCUSSION THREAD

**Discussion by brandondev1** [amzn/selling-partner-api-models#3734] [General]
Is there a way to get the recommended quantity information for FBA inventory that is available in the Selling Partner Portal.  It shows current inventory qty, recommended qty, supply lead time, and the recommended order date.  

![Capture](https://github.com/amzn/selling-partner-api-docs/assets/18074719/6e781fc0-bbcf-4857-8d60-2b2f38b45d94)

**Comment by stefnats** [comment]
Not sure if this helps, but maybe one of these reports shows the information.
The image is a bit unclear and small, so this is a long shot.

https://developer-docs.amazon.com/sp-api/docs/fba-inventory-reports-attributes#get_fba_inventory_planning_data

But also maybe you can calculate it yourself without API calls. 
you maybe know the sales and the quantity that is in amazon warehouses. with that you can plan your fba shipments.
